Assets/MRTK/Core/Utilities/BuildAndDeploy/UwpAppxBuildTools.cs

```diff
@@ -229,13 +234,9 @@ private static bool UpdateVSProj(IBuildInfo buildInfo)
         {
             // For ARM64 builds we need to add ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch
```

Comment:
Side note, but I believe this bug was fixed in https://github.com/microsoft/msbuild/pull/4389 and I don't think we actually need to write this anymore (as long as the build system VS installations have updated to the fixed version)

Reply:
But gonna merge this and take it one step at a time